packages/closer/pages/bookings/create/dates.tsx

```
@@ -16,7 +16,7 @@ import HeadingRow from '../../../components/ui/HeadingRow';
 import ProgressBar from '../../../components/ui/ProgressBar';
 
 import dayjs from 'dayjs';
-import { NextPageContext } from 'next';
+import { NextApiRequest, NextPageContext } from 'next';
```

ESLint

[error] 'NextApiRequest' is defined but never used.

(unused-imports/no-unused-imports)